lib/managers/folders.js

```diff
@@ -301,6 +303,53 @@ Folders.prototype.deleteMetadata = function(folderID, scope, template, callback)
 	this.client.del(apiPath, null, this.client.defaultResponseHandler(callback));
 };
 
+/**
+ * Retrieves a folder that has been moved to the trash
+ *
+ * @param  {type} folderID  - The ID of the folder being requested
+ * @param {?Object} qs - Additional options can be passed with the request via querystring. Can be left null in most cases.
+ * @param  {type} callback  - Passed the folder information if it was acquired successfully
+ * @returns {void}
+ */
+Folders.prototype.getTrashedFolder = function(folderID, qs, callback) {
+	var params = {
+		qs: qs
+	};
+
+	var apiPath = urlPath(BASE_PATH, folderID, '/trash/');
+
+	// Handle Special API Response
+	this.client.get(apiPath, params, function(err, response) {
+
+		// Error - No successful request could be made
+		if (err) {
+			callback(err);
+			return;
+		}
+
+		/* eslint-disable callback-return */
+		switch (response.statusCode) {
```

Comment:
In this case, we use "unexpected" to mean any failure case.  This is to differentiate from some endpoints which might have multiple "expected" responses (e.g. a 200 or 302 might both be considered "successful").  However, you probably would not expect to get a 404 here, since that would mean your call was in error.